rust/src/api.rs

```diff
@@ -16,6 +16,14 @@ pub fn init_wallet(network: Network) -> Result<()> {
     wallet::init_wallet(network)
 }
 
-pub fn get_balance() -> anyhow::Result<Balance> {
+pub fn get_balance() -> Result<Balance> {
     Ok(Balance::new(wallet::get_balance()?.confirmed))
 }
+
+pub fn get_seed_phrase() -> Result<Vec<String>> {
+    let seed_phrase = wallet::get_seed_phrase()?;
+
+    // The flutter rust bridge generator unfortunately complains when wrapping a ZeroCopyBuffer with
+    // a Result. Hence we need to copy here (data isn't too big though, so that should be ok).
```

Comment:
yeah, it's sent over once, so it's not a problem at all.

sometimes boxing (ie. wrapping in a box, like : `Result<Box<Type>>`) can help, as it just passes a pointer to  data already on the heap, but I don't think it's worth it here